app/resources/v1/users.js

```diff
@@ -370,7 +294,7 @@ exports.get = async function (req, res) {
     // seems odd to have these two functions chained together, when we
     // have an ORM that can retrive the user details. is this code still used?
     const result = await findUserById(userId)
-    handleFindUser(result)
+    res.status(200).send(asUserJson(result))
```

Comment:
I think this is making some more sense. I wonder whether we need asUserJson though? 

This could be out of scope for this PR, but it appears to just be an extra step, _maybe_ returning some fields we don't need, but generally not seeing what value were getting when this may just be some old technical debt. 

I didn't test this, but I think that 296-297 could just be written as something like: 
```
const user= await User.findByPk(userId).toJSON()
```
Which would all be using built in sequelize stuff. 

https://github.com/streetmix/streetmix/blob/main/lib/util.js#L30
Has this funny little function to make sure 'USER' is the first order in roles. I wonder why we need this and if the requirement can be removed. If not, it could be added as a custom getter to the user model which would then be called using the above functions (again, not tested)

Reply:
Right, so `asUserJson` is a utility function that has its roots in our previous MongoDB implementation and does some additional meddling with the user object before sending it off to the client. As to whether or not any of that is necessary or worth refactoring at the moment, these are worthy questions to ponder, but certainly not in scope for this PR.

If you'd like, please extract this out to an issue and I'd appreciate if you would consider tackling this yourself.

Reply:
I originally asked to clarify whether the comment was applied to the Twitter client functionality, but since it's not, I'll leave the comment as-is.

Reply:
stubbed out in https://github.com/streetmix/streetmix/issues/2276